gce.py

```diff
@@ -215,8 +215,12 @@ def node_to_dict(self, inst):
                 md[entry['key']] = entry['value']
 
         net = inst.extra['networkInterfaces'][0]['network'].split('/')[-1]
+        # If there is no public IP listed, set the public_ip field to a null.
         if inst.public_ips.__len__() == 0:
           inst.public_ips=[None]
+
+        # Use private IPs if we're told to, or we've got no choice.
+        if os.environ.has_key("ANSIBLE_USE_PRIVATE_SSH") or inst.public_ips==[None]:
```

Comment:
I'm not sure about the environment variable. Do we ever want to turn it off?

If not, we shouldn't support doing so because it will remain untested.

If we do, then I think we should add another key to the config file that it already uses, because it's easier to understand what it's being used by.